parts/dcosagentvars.t

```diff
@@ -26,4 +26,5 @@
     "{{.Name}}LbIPConfigID": "[concat(variables('{{.Name}}LbID'),'/frontendIPConfigurations/', variables('{{.Name}}LbIPConfigName'))]", 
     "{{.Name}}LbIPConfigName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]", 
     "{{.Name}}LbName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]",
-{{end}}
\ No newline at end of file
+{{end}}
+    "dcosBootstrapURL": "[parameters('dcosBootstrapURL')]", 
```

Comment:
Sorry, I made a mistake. Put the variable to the master vars will be OK. I will change it. Thanks!